account_invoice_reorder_lines/README.rst

```diff
@@ -0,0 +1,46 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
```

Comment:
Please add here below another row of equal signs, see https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst ass reference.